pkg/webui/console/store/reducers/gateway.js

```diff
@@ -0,0 +1,110 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {
+  GET_GTW,
+  GET_GTW_SUCCESS,
+  GET_GTW_FAILURE,
+  START_GTW_STATS,
+  UPDATE_GTW_STATS,
+  UPDATE_GTW_STATS_SUCCESS,
+  UPDATE_GTW_STATS_FAILURE,
+  UPDATE_GTW_STATS_UNAVAILABLE,
+  STOP_GTW_STATS,
+} from '../actions/gateway'
+
+const statsDefaultState = {
+  fetching: false,
+  available: true,
```

Comment:
So why is it `available: true` by default?

Reply:
why not? `available: false` means that the stats indicator is hidden which is the case only when the console has no gs in the cluster. all other cases display the stats, so we either set to `true` by default or on every other action which is not `UPDATE_GTW_STATS_UNAVAILABLE`